2.wave3_data/1.cellprofiler_quality_control/README.md

```diff
@@ -0,0 +1,34 @@
+# Create LoadData csvs to create maximum projection images
+
+In this module, we create two LoadData csv and CellProfiler pipelines for performing maximum projection on the images and saving them (the images will be named p03 where 3 is the last z-plane).
```

Comment:
just confirming, the `3` doesn't imply only 3 images? It implies 4 images because of zero indexing? In other words, we know there are 3 images in the full z stack, and I want to confirm that we're not just taking the third image (instead of the max projected image).

Reply:
Based on Erin's response:

"Using our standard pipeline, projected images will share the file name of the last z-plane e.g. source images z1-z8 will create a projection called z8 so delete all non-z8 images."

Essentially the `zproj.cppipe` pipeline names the max projected images as the last z-plane so in our case we have 3 z-planes (p01-p03) so our images contain the last z-plane p03.

Reply:
got it!! Yes, this makes perfect sense. It is probably worth adding some of that description in the README.

Reply:
I will add it in now before merging!